md-commons/src/main/java/com/wipro/ats/bdre/md/beans/table/AppDeploymentQueue.java

```diff
@@ -31,13 +31,13 @@
     public AppDeploymentQueue(){
     }
 
-    public AppDeploymentQueue(Long adqId, Integer processId, String username, String appDomain, String appName, Short adqState, Integer pageSize, Integer page, Integer counter) {
-        this.adqId = adqId;
+    public AppDeploymentQueue(Long appDeploymentQueueId, Integer processId, String username, String appDomain, String appName, Short appDeploymentQueueStatus, Integer pageSize, Integer page, Integer counter) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 9 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00107)